bnb/getBNBData

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+debug = False
+
+import io, os, re, requests, sys, time, zipfile
```

Comment:
Do this on separate lines ([PEP8 style](https://pep8.org/#imports)). Maybe we can simply make sure that the code passes `pycodestyle` w/o any errors/warnings.

Reply:
This has been addressed by fde2af2